alwaysontop/alwaysontop.js

```diff
@@ -0,0 +1,41 @@
+const {
+    api,
+    ondblclick,
+    onload,
+    onbeforeunload
+} = window.alwaysOnTop;
```

Comment:
Is there a guarantee window.alwaysOnTop will be defined by the time alwaysontop.js is running?